custom-tests.yaml

```diff
@@ -766,6 +766,37 @@ api:
       var instance = new FetchEvent('fetch', { request: req });
   FileReader:
     __base: var instance = new FileReader();
+  FileSystem:
+    __base: >-
+      window.webkitRequestFileSystem(TEMPORARY, 8*1024 /* 8KB */, function(fs) {
```

Comment:
Gecko also has an unprefixed `FileSystem` interface which you can get from `FileSystemEntry` objects. Can you test if there's a way to get to a `FileSystem` instance in Firefox?